### PR TITLE
Migrate more scss to styled components

### PR DESCRIPTION
**Pull request by mmahalwy**
### Why?
Firstly, we have been using a hacky library called `webpack-isomorphic-tools` to get styles working on the server. There are alternatives now, such as getting webpack to serve server files but regardless this is a great opportunity to transition into css-in-js. What's most powerful about it is code-splitting CSS comes for free since it's in the JS bundle. Additionally, we will continue to make more DLS components that are easily customizable thanks to styled-components

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32960

File: src/components/Audioplayer/RepeatDropdown/index.js [hunk 1]
```diff
@@ -9,7 +10,37 @@ import { intlShape, injectIntl } from 'react-intl';
...
+`;
+
+const Pill = styled(NavItem)`
+  a{
+    padding: 10px 15px;
+  }
+`;
+
+const disabled = css`
+opacity: 0.5;
```
Comment (sabeurthabti) [inline src/components/Audioplayer/RepeatDropdown/index.js]: indent, if you use VS Code, it has support for styled-components

File: src/components/SurahInfo/index.js [hunk 2]
```diff
@@ -1,37 +1,100 @@
 import React, { PropTypes } from 'react';
 import * as customPropTypes from 'customPropTypes';
+import styled from 'styled-components';
+import { lighten } from 'polish';
```
Comment (sabeurthabti) [inline src/components/SurahInfo/index.js]: do you mean `polished`?

File: src/components/Audioplayer/RepeatDropdown/index.js (hunk 1 above)
Comment (mmahalwy) [reply src/components/Audioplayer/RepeatDropdown/index.js]: I do. Hmm, not sure why didn't indent. Will fix

**Comment by mmahalwy** [comment]
@sabeurthabti good catch. Weird my webpack didn't scream.

**Comment by mmahalwy** [comment]
@sabeurthabti PTAL. Fixed

**Comment by mmahalwy** [comment]
rebuild

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32962

**Pull request merged**